### ISSUE TITLE
MediaQuery.queries array seems "overcrowded"

### ISSUE THREAD

**Issue by aj1991** [zurb/foundation-sites#11540]
## What is missing?
By resizing window the Foundation.MediaQuery.queries array will be filled with a lot of queries. They vary by changing breakpoints by resizing the window (sometimes more, sometimes less). I was expecting that there are only my defined "css" breakpoints available.

![snip_20181017095355](https://user-images.githubusercontent.com/9250626/47071663-6619ea80-d1f4-11e8-8605-bc6520cb33ee.png)

## What is wrong now?
I dont think that there should be an array of 217 queries. The amount should not exceed my amount of css breakpoints.

## Your Environment
- Foundation 6.4.3
- Chrome 69 
- Windows 10

## Checklist
- [X] I have read and follow the CONTRIBUTING.md document.
- [X] here are no other issues similar to this one.
- [X] The issue title and template are correctly filled.

**Comment by ncoden** [comment]
Hi @aj1991 👋, thank you for the issue,

This is not normal. But does is lead to any visible bug in components or with `MediaQuery.is()` ? Thank you.

**Comment by aj1991** [comment]
At that moment, I didn't receive any error - I just found this by debugging another issue ;)

**Comment by SassNinja** [comment]
The reason for those duplicate queries is that `MediaQuery._init()` gets called multiple times (from different plugins).

I've submitted a PR (https://github.com/zurb/foundation-sites/pull/11543) which fixes this by making sure it populates the `queries` only once.
